pygsti/modelpacks/stdtarget.py

```diff
@@ -42,127 +42,6 @@ def _get_cachefile_names(std_module, param_type, simulator, py_version):
     raise ValueError("No cache files used for param-type=%s" % param_type)
 
 
-# XXX is this used?
```

Comment:
This was used for modelpack generation. Fine to remove but note TermForwardSim caching for model packs

Reply:
To clarify, @sserita, would you like the source code to include a comment of some kind given that we'll be removing this?